tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+  "compilerOptions": {
```

Comment:
nit: lexicographical sort of properties, noting however this might be an antipattern in js/ts world so happy to hear from any other reviewers if this would be suitable.
Yet again, please don't consider this a blocker for approval/merge

Reply:
Hi @JayRovacsek, The `tsconfig.json` is based on the order in a default `tsconfg.json` with a few things added. I find it a little easier to read with the rough order it's in as it's got things where I've come to expect them but I wouldn't call it standard. I'm happy to take any instruction on how to order this.

Reply:
Sounds good - let's consider this comment resolved 👍